crates/store/re_data_source/src/data_source.rs

```diff
@@ -261,10 +261,7 @@ impl LogDataSource {
 
                 spawn_future(async move {
                     if let Err(err) = stream_partition.await {
-                        re_log::warn!(
-                            "Error while streaming {uri}: {}",
-                            re_error::format_ref(&err)
-                        );
+                        re_log::warn!("Error while streaming: {}", &err);
```

Comment:
removed because the error already contains the uri? Why remove `format_ref`, bad formatting?